### PR TITLE
Get stories

### LINKED ISSUE #12: Create Endpoint: GET /stories
Endpoint is responsible for fetching all the stories from the DB

### LINKED ISSUE #15: Create Endpoint: GET /story/{id}
This endpoint returns a story by it's ID

### PR DESCRIPTION
**Pull request by aspensantos**
Closes #12 
Closes #15

### CODE REVIEW

**Comment by s278gupt** [comment]
lgtm! please rebase on https://github.com/uwblueprint/shoe-project/pull/41 and tophat :)

**Comment by dhillondeep** [comment]
@m-penny #41 is merged now!

File: restapi/stories.go [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+}
+
+func (api api) ReturnStoryByID(w http.ResponseWriter, r *http.Request) render.Renderer {
+	var story models.Story
+	id := chi.URLParam(r, "storyID")
+	err := api.database.Where("id=?", id).First(&story).Error
+
+	if err != nil {
+		if err == gorm.ErrRecordNotFound {
+			return rest.ErrNotFound("Could not find story with ID: " + id)
```
Comment (dhillondeep) [inline restapi/stories.go]: it is a nitpick but try using `fmt.Sprintf()` to structure strings

File: restapi/stories.go [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+	"github.com/go-chi/render"
+	"github.com/uwblueprint/shoe-project/internal/database/models"
+	"github.com/uwblueprint/shoe-project/restapi/rest"
+	"gorm.io/gorm"
+)
+
+func (api api) ReturnAllStories(w http.ResponseWriter, r *http.Request) render.Renderer {
+	var stories []models.Story
+	err := api.database.Find(&stories).Error
+
```
Comment (dhillondeep) [inline restapi/stories.go:16]: usually in go codebases the line generating the error and error check are joined together and there is no space

**Review by dhillondeep — APPROVED** [review]

**Pull request merged**